gradle/profile_prod.gradle

```diff
@@ -1,44 +1,34 @@
-apply plugin: 'org.springframework.boot'
-apply plugin: 'com.gorylenko.gradle-git-properties'
-
-ext {
-    logbackLoglevel = "INFO"
```

Comment:
apply the same changes as in profile_dev file

Reply:
done